src/objects/Allocation__c.object

```diff
@@ -51,6 +51,11 @@
     <enableSearch>true</enableSearch>
     <enableSharing>true</enableSharing>
     <enableStreamingApi>true</enableStreamingApi>
+    <fieldSets>
+        <description>Fields added to this field set will be displayed as editable columns in the Manage Allocations table</description>
+        <fullName>ManageAllocationsTableFields</fullName>
+        <label>Manage Allocations Table Fields</label>
```

Comment:
@sam-knox Can you review the label and description that will be used to configure additional fields on the Manage Allocation page.

Reply:
@mattwashington - Since you're going to do the documentation on this one, can you review the label for Andrew here?